Spreadsheet/index.html

```diff
@@ -33,223 +29,220 @@
                 <div id="table-input">
                     <input id="fxInput" type="text" value="">
                 </div>
-
             </div>
 
             <div class="table">
-                <div id="numbers">
-                    <div class="box" id="0"></div>
-                    <div class="box" id="1">1</div>
-                    <div class="box" id="2">2</div>
-                    <div class="box" id="3">3</div>
-                    <div class="box" id="4">4</div>
-                    <div class="box" id="5">5</div>
-                    <div class="box" id="6">6</div>
-                    <div class="box" id="7">7</div>
-                    <div class="box" id="8">8</div>
-                    <div class="box" id="9">9</div>
-                    <div class="box" id="10">10</div>
+                <div id="rowHeaders">
```

Comment:
`rowHeaders` seem like a more direct description of what this element represents. `numbers` is more of a description of what type of data it contains (so less informative for the reader)